<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownLoad</class>
 <widget class="QMainWindow" name="DownLoad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DownLoad</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QGroupBox" name="groupBox_download">
      <property name="title">
       <string>Fimware Download</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_file">
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>no file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_watting">
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>watting</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="m_progressBar">
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>16</height>
             </size>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_1" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="Button_Loadfile">
            <property name="text">
             <string>Load File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Button_Download">
            <property name="text">
             <string>DownLoad</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="checkBox_mav">
              <property name="text">
               <string>MAV</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_mul">
              <property name="text">
               <string>Mul</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_Mul" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>200</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="Form_Download" name="widget_D1" native="true"/>
          </item>
          <item>
           <widget class="Form_Download" name="widget_D2" native="true"/>
          </item>
          <item>
           <widget class="Form_Download" name="widget_D3" native="true"/>
          </item>
          <item>
           <widget class="Form_Download" name="widget_D4" native="true"/>
          </item>
          <item>
           <widget class="Form_Download" name="widget_D5" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="Form_Download" name="widget_D6" native="true"/>
          </item>
          <item>
           <widget class="Form_Download" name="widget_D7" native="true"/>
          </item>
          <item>
           <widget class="Form_Download" name="widget_D8" native="true"/>
          </item>
          <item>
           <widget class="Form_Download" name="widget_D9" native="true"/>
          </item>
          <item>
           <widget class="Form_Download" name="widget_D10" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>664</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionSimplified_Chinese"/>
   </widget>
   <addaction name="menuLanguage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionSimplified_Chinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Simplified Chinese</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Form_Download</class>
   <extends>QWidget</extends>
   <header location="global">form_download.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
